apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -87,24 +86,6 @@ describe('FinishStudentAccount', () => {
     screen.getByText(locale.go_to_my_account());
   });
 
-  it('does not render state question if user is not detected in the U.S.', () => {
```

Comment:
Why is this removed?

Reply:
Oh whoops I think a merge conflict caused this! Thanks for catching I'll put this back in!